DamgleDamgle_iOS/DamgleDamgle_iOS/Home/HomeViewController.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-class HomeViewController: UIViewController, BaseViewController {
+class HomeViewController: UIViewController {
     
     @IBOutlet private weak var currentAddressLabel: UILabel!
     @IBOutlet private weak var monthlyPaintingBGView: UIView!
```

Comment:
```suggestion
    @IBOutlet private weak var monthlyPaintingBGView: UIView! {
        didSet {
            monthlyPaintingBGView.layer.cornerRadius = 8
        }
    }
```

아레쪽에 setUpView 대신에 이렇게 하는 것도 방법이겠네요!